### ISSUE TITLE
Feature: Add the ability to configure UI notifications by providing the desired level of notifications

### ISSUE THREAD

**Issue by utkarshg6** [MASQ-Project/MASQ-Node-issues#582]
There are different log levels. We may want to use some of which that already exist.

We have the following log types as in #349:
1. Error
2. Warning
3. Info
4. Debug [ Dev Logs ]
5. Trace [ Dev Logs ]

There is an idea that we can display _info level_ into different levels.

The user should be able to configure the type of notifications she/he might want to receive, and this configuration can be 
customized during both _startup_ and _runtime_  configuration.

This card is closely related to card #349. We may like to proceed with this card once #349 is completed. 

Possible design

1) It'll probably use a global variable to configure the type of notification.
2) Ability to configure at runtime also (possibly another card).
3) Read that variable without expensive computation. [Critical obstacle]
4) We will check a condition with this variable extremely often. So, we need an optimized way to read it. 
5) If we have to use a Mutex to avoid unsafe code, make sure we don't lock the Mutex every so often but only after a certain amount of time to see if the value has been updated by the user in the meantime, otherwise we just use a reference to the value inside the Mutex.
5) We might introduce this parameter into the database and will provide this parameter at tools for displaying configuration.

The design may be changed as per the requirement.

Last and not least, disscus with the GUI team and be ready to discard this card if it turns out to be an unneeded card from their perspective.

**Comment by anonmate** [comment]
From a UI perspective, I'm not sure we are going to need to set this.

Granted I don't know what error, warn, and info means to you guys but I imagine we'll always need Error, Warning, and Info logs as JSON through websockets so we can act on them accordingly. Debug and Trace I guess appears in the node logs anyway, which we already show in the settings panel of the UI.

I hope we won't build too much logic around error messages. I'm hoping to handle them in a standard way like:

• Errors - Node probably shuts down and needs a user handled restart (it depends what error means to you guys, though - this might not be the way to handle all errors, but I assume they mean that the node can't continue forward with something and needs intervention)
• Warnings - Show notifications if relevant to the user (payments, no routing found)
• Info - Depending on what info commands there are, show as notification.

It would be handy to have documentation of all of these possible errors, warns, and info logs that could happen so we can decide what to do with them in the UI. 

Remember, we can action a lot with the UI when we have error messaging. So for errors that happen we can automate their solutions in a lot of cases. But each time we do that, we should discuss with the Node team (or just @utkarshg6 @bertllll) so that we're not implementing solutions for errors that really should be fixed in Node, and not handled just by restarting or some other automated change of configuration in the UI.

The point is to try and keep most if not all the logic within Node and not delegate out to the UI to handle errors because it's easy. But if it makes more sense to handle errors with the UI, then we can.
